tests/Sabre/Xml/ServiceTest.php

```diff
@@ -231,10 +231,20 @@ function testMapValueObject() {
 
         $this->assertEquals($expected, $order);
 
-        $writtenXml = $orderService->write('{http://sabredav.org/ns}order', $order);
+        $writtenXml = $orderService->writeValueObject($order);
```

Comment:
Shouldn't we test both api calls and dont assume impl details (that those are equivalent). We should assert those are equivalent instead